src/DocChecks.jl

```diff
@@ -88,6 +88,20 @@ sigs(::Any) = Type[Union{}]
 # Julia code block testing.
 # -------------------------
 
+# helper to display linerange for error printing
+function find_codeblock_in_file(code, file)
+    content = read(Base.find_source_file(file), String)
+    content = replace(content, "\r\n" => "\n")
```

Comment:
We can do this to make this work with CLRF since the number of lines does not change.